business/istio_config.go

```diff
@@ -152,10 +161,10 @@ func (in *IstioConfigService) GetIstioConfigList(criteria IstioConfigCriteria) (
 		return models.IstioConfigList{}, err
 	}
 
-	errChan := make(chan error, 19)
+	errChan := make(chan error, 20)
```

Comment:
Should this be a 21? You adding to new calls, so you expect up to two more errors in that channel.

Reply:
I think it was out-of-sync and I've adjusted.
![image](https://user-images.githubusercontent.com/1662329/81315081-26427e80-908a-11ea-8f7f-3b57bccbd77e.png)

There are 22 goroutines there, but only errChan is passed to 20.

I think that I don't miss it but let me know if I missed something perhaps I coded it fast.